k8s/upgrades/jiva-0.6.0-0.8.1/README.md

```diff
@@ -0,0 +1,151 @@
+OpenEBS 0.8.1 auto installs a default Jiva Storage Pool and a default Storage Class named `default` and `openebs-jiva-default` respectively. If you have a storage pool named `default` created in earlier version, you will have to re-apply your Storage Pool after the upgrade is completed.
+
+Before upgrading the OpenEBS Operator, check if you are using a storage pool named `default` which will conflict with default jiva pool installed with OpenEBS 0.8.1:
+```
+./pre_upgrade.sh <openebs-namespace>
+```
+
+#### Storage Classes
+OpenEBS supports specified Storage Policies in Storage Classes. The way storage policies are specified has changed in 0.7.x. The policies will have to be specified under metadata instead of parameters. 
+
+For example, if your storage class looks like this in 0.6.0:
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: openebs-mongodb
+provisioner: openebs.io/provisioner-iscsi
+parameters:
+  openebs.io/storage-pool: "default"
+  openebs.io/jiva-replica-count: "3"
+  openebs.io/volume-monitor: "true"
+  openebs.io/capacity: 5G
+  openebs.io/fstype: "xfs"
+```
+
+There is no need to mention the volume-monitor and capacity with 0.7.0. The remaining policies like storage pool, replica count and the fstype should be specified as follows:
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: openebs-mongodb
+   annotations:
+    cas.openebs.io/config: |
+      - name: ReplicaCount
+        value: "3"
+      - name: StoragePool
+        value: default
+      - name: FSType
+        value: "xfs"
+provisioner: openebs.io/provisioner-iscsi
+```
+
+Make edits to your Storage Class YAMLs - delete them and add them back. A delete and re-apply is required since updates to Storage Class parameters are not possible. 
+
+If you are using `ext4` for FSType, you could use the following script to upgrade your StorageClasses. 
+```
+./upgrade_sc.sh
+```
+
+Alternatively, you can skip this step and re-apply your StorageClasses as per the 0.7.0 volume policy specification. 
+
+**Important Note: StorageClasses have to updated prior to provisioning any new volumes with 0.7.0.**
+
+## Step 1: Upgrade the OpenEBS Operator
+
+### Upgrading OpenEBS Operator CRDs and Deployments
+
+The upgrade steps vary depending on the way OpenEBS was installed, select one of the following:
+
+#### Install/Upgrade using kubectl (using openebs-operator.yaml )
+
+**The sample steps below will work if you have installed openebs without modifying the default values in openebs-operator.yaml. If you have customized it for your cluster, you will have to download the 0.7.0 openebs-operator.yaml and customize it again**
+
+```
+#If Upgrading from 0.5.x, delete older operator. 
+# Starting with OpenEBS 0.6, all the components are installed in namespace `openebs`
+# as opposed to `default` namespace in earlier releases. 
+kubectl delete -f https://raw.githubusercontent.com/openebs/openebs/v0.5/k8s/openebs-operator.yaml
```

Comment:
lets remove this section as we are upgrading from 0.6

Reply:
done